scripts/buildx.sh

```diff
@@ -151,7 +162,7 @@ elif docker info; then
         --platform="linux/$ARCH" \
         --env ACLOCAL_PATH="/usr/share/aclocal:/usr/local/share/aclocal" \
         "$BUILDER" \
-        bash -euc "$CMD_RUN"
+        bash -euc "git config --global --add safe.directory /emqx && $CMD_RUN"
```

Comment:
Suggestion: since dependencies might complain about other unsafe git directories:

```suggestion
        bash -euc "git config --global --add safe.directory \* && $CMD_RUN"
```

Reply:
the build scripts inside container does not seem to complain.